package.json

```diff
@@ -15,38 +16,41 @@
     "publish:next": "npm publish --access public --tag next"
   },
   "peerDependencies": {
-    "@angular/common": "^10.0.0 || ^11.0.0",
-    "@angular/platform-server": "^10.0.0 || ^11.0.0",
+    "@angular/common": "^12.0.0",
+    "@angular/platform-server": "^12.0.0",
```

Comment:
Can we remain backward compatible as well?

Reply:
Hey! Is there any reason we'd want to make it backward compatible? This will bring some additional code complexity which I don't think we would benefit from.

Since the `@nguniversal` schema has been changed in v12, should we check the Angular version and provide different schema when running external schematics?